packages/api-docs-builder/buildApi.ts

```diff
@@ -211,8 +211,7 @@ async function run(argv: yargs.ArgumentsCamelCase<CommandOptions>) {
               // Box's demo isn't ready
               // Container's demo isn't ready
               // Grid has problem with react-docgen
-              // Stack has problem with react-docgen
-              component.filename.match(/(Box|Container|ColorInversion|Grid|Stack)/)) ||
+              component.filename.match(/(Box|Container|ColorInversion|Grid)/)) ||
```

Comment:
I fixed the Stack generation in #36785 but forgot to enable its page generation. It's why it's a 404: https://mui.com/joy-ui/api/stack/.